uniffi_bindgen/src/bindings/python/gen_python.rs

```diff
@@ -53,7 +53,7 @@ mod filters {
             FFIType::Float64 => "ctypes.c_double".to_string(),
             FFIType::RustCString => "ctypes.c_voidp".to_string(),
             FFIType::RustBuffer => "RustBuffer".to_string(),
-            FFIType::RustError => "RustError".to_string(),
+            FFIType::RustError => "POINTER(RustError)".to_string(),
```

Comment:
To my untrained eye, this doesn't look like Python.

Reply:
Nice catch, this at least needs `ctypes.POINTER(RustError)` otherwise the imports won't work out correctly.